cirq/linalg/decompositions.py

```diff
@@ -854,3 +857,98 @@ def kak_decomposition(unitary_object: Union[np.ndarray, 'cirq.SupportsUnitary'],
         global_phase=g * inner_cannon.global_phase,
         single_qubit_operations_before=(b1, b0),
         single_qubit_operations_after=(a1, a0))
+
+
+def kak_vector(unitary: np.ndarray):
+    r"""Compute the KAK vectors of a two qubit unitary.
+
+    Any 2 qubit unitary may be expressed as
+
+    $$ U = k_l A k_r $$
+    where $k_l, k_r$ are single qubit (local) unitaries and
+    $$ A = \exp( i * \sum_{j=x,y,z} k_j \sigma_{j,0}\sigma{j,1} ) $$
+
+    where $(k_x,k_y,k_z)$ is in the Weyl chamber, a tetrahedron with corners
+    $$(0,0,0), (\pi/2,0,0), (\pi/4,pi/4,0), (\pi/4,\pi/4,\pi/4)$$
+
+    The vector entries are ordered such that
+    $$ \pi/2 - k_y \geq k_x \geq k_y \geq z \geq 0 $$
+
+    Args:
+        unitary: Shape (...,4,4) unitary numpy.ndarray.
+
+    Returns:
+        The KAK vector of the unitary, output shape (...,3).
+
+    References:
```

Comment:
Add an `Examples` section

```
Examples:
    >>> cirq.kak_vector(np.eye(4))
    [0, 0, 0]

    >>> cirq.kak_vector([cirq.unitary(cirq.CZ), cirq.unitary(cirq.ISWAP)])
    [[1, 0, 0], [1, 1, 0]]
```

Be sure to use the `>>> line` then `result` style, as that way the example will be tested by doctest.

Reply:
Done.